packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -435,6 +429,7 @@ export class AwsCustomResource extends cdk.Construct implements iam.IGrantable {
  */
 function awsSdkToIamAction(service: string, action: string): string {
   const srv = service.toLowerCase();
+  const awsSdkMetadata = getAwsSdkMetadata();
```

Comment:
We should cache this call, otherwise we are going to read and parse for every action in every `AwsCustomResource`

Reply:
Updated in revision.